src/backend/routes/api.ts

```diff
@@ -13,25 +13,19 @@ export const konfigurerApi = (app: Express): Express => {
     app.use(`${basePath}api/soknad`, express.json({ limit: '5mb' }) as RequestHandler);
     app.use(`${basePath}api/soknad`, erklaeringInterceptor);
     app.use(`${basePath}api/soknad`, escapeBody);
-    if (!erLokalt()) {
```

Comment:
Hva er grunnen til at vi trenger å gjøre dette også lokalt? 😊

Reply:
Vi trenger strengt tatt ikke gjøre dette, for vi kunne fortsatt med token i cookie, men med token i Authorization header blir det mer likt som preprod og prod. Hvis vi skulle fortsatt med cookie måtte vi også gjort noen endringer i configen til backend da det sluttet å fungere lokalt når jeg fjernet selvbetjening-issuer derfra.

```
no.nav.security.jwt:
  issuer:
    selvbetjening:
      discoveryurl: http://localhost:${mock-oauth2-server.port}/selvbetjening/.well-known/openid-configuration
      accepted_audience: aud-localhost
      cookie_name: localhost-idtoken // denne linja sørget for at vi plukket opp token fra cookie.
    tokenx:
      discovery-url: http://localhost:${mock-oauth2-server.port}/tokenx/.well-known/openid-configuration
      accepted-audience: familie-app
```